objects/user.php

```diff
@@ -1272,5 +1298,16 @@ public function userIsTempBanned()
 
 		return $tempBan > time();
 	}
+
+	/* 
+	 * Determine style theme user is using

```

Comment:
Mention in the comment that this will be null in some cases so additional handling is needed for those cases, or just adjust this to return light mode if the query didn't return anything and remove the handling from the calling functions.